app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,8 @@
+module AdminHelper
+  def admin_page?
+    url = request.original_url.split("/")
+    return true if url[3] == "admin" || url[4] == "admin"
```

Comment:
```suggestion
    url[3] == "admin" || url[4] == "admin"
```
chỉ cần dòng này là xong. cần gì thêm dòng 5